src/scss/variables.scss

```diff
@@ -25,3 +25,6 @@ $navbar-height: 75px;
 
 $font-primary: $font-dm-sans;
 $font-header: $font-poppins;
+
+$transition-short: 0.15s;
+$transition-long: 0.5s;
```

Comment:
Not sure if that's the right spot for them. They make sense to align the duration of the transitions of the border and the button, but they seem to be more local in nature. This would be a global scope though.

These durations has been used somewhat consistently, but I don't see the value in unifying duration length strictly unlike colors, border radiuses, fonts etc.

Reply:
These are the only two durations ever used. `0.15s` for any kind of color transitions for hover effects, and `0.5s` for things like the playlist card image scaling up. That's consistent enough to me. I also changed all transitions throughout the project to use these variables, that way we can stay consistent in the future.

Reply:
Generally, I still don't see the value in that unless they are maybe rather named as "color-transition" or similar. Might even encourage choosing this value for no particular reason.